src/types/wallet.ts

```diff
@@ -41,6 +41,7 @@ export type StoredWallet = {
   id: ActiveWallet['id']
   name: ActiveWallet['name']
   encrypted: string
+  lastUse: Date
```

Comment:
It is recommended to store dates in Redux as strings or numbers (timestamps): https://github.com/reduxjs/redux-toolkit/issues/456

See for example how we handle the `lastUsed` prop of the `AddressState` type: https://github.com/alephium/desktop-wallet/blob/next/src/types/addresses.ts#L60